pkg/scheduler/util/utils.go

```diff
@@ -163,3 +166,20 @@ func ClearNominatedNodeName(cs kubernetes.Interface, pods ...*v1.Pod) utilerrors
 	}
 	return utilerrors.NewAggregate(errs)
 }
+
+// NewInformerFactory creates a SharedInformerFactory and initializes a scheduler specific
+// in-place podInformer.
```

Comment:
I guess this is open to other `Informer` beside `podInformer`?

Reply:
Let's see. But now it just sticks to scheduler specific podInformer.